consensus/src/processes/transaction_validator/transaction_validator_populated.rs

```diff
@@ -120,7 +120,7 @@ impl TransactionValidator {
         Ok(())
     }
 
-    fn check_scripts(&self, tx: &impl VerifiableTransaction) -> TxResult<()> {
+    pub fn check_scripts(&self, tx: &impl VerifiableTransaction) -> TxResult<()> {
```

Comment:
Good observation :) 

I removed the code that was using it because it was in the comments.  This fn needs to be pub to be accessible externally so that an external module can load and use it to test/validate scripts.

I had a problem with scripts during tx creation (turned out this was due to BIP44 account index fn override that was mismanaged during the account trait refactoring), but to identify the source (as server was reporting a script error) I needed to short-circuit the processing and check scripts directly in the wallet module (generator) during tx creation.

So as a matter of an API or potential use in wallet account unit tests in the future this fn needs to be pub.